kolibri/core/assets/src/views/immersive-app-bar/index.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+
+  <ui-toolbar
+    :title="appBarTitle"
+    textColor="white"
+    type="colored"
+    :navIcon="icon"
+    :showIcon="showIcon"
+    @nav-icon-click="$emit('navIconClick')"
+    :style="{ height: height + 'px' }"
+  >
+    <div v-if="hasRoute" slot="icon">
+      <router-link :to="route">
+        <!-- TODO add aria label? -->
+        <ui-icon-button
+          type="flat"
+          @click="$emit('navIconClick')"
+          :icon="icon"
+          class="icon"
+        />
+      </router-link>
+
+    </div>
+  </ui-toolbar>
+
+</template>
+
+
+<script>
+
+  import uiToolbar from 'keen-ui/src/UiToolbar';
+  import uiIconButton from 'keen-ui/src/UiIconButton';
+  import { validateLinkObject } from 'kolibri.utils.validators';
+
+  export default {
+    name: 'immersiveAppBar',
```

Comment:
`immersiveActionBar` would be a more appropriate name?

Reply:
I thought the term "Action Bar" was deprecated, so I tried to follow material in this case. You disagree?

> The app bar, formerly known as the action bar in Android, is a special kind of toolbar that’s used for branding, navigation, search, and actions.

https://material.io/guidelines/layout/structure.html#structure-app-bar